deploy/deploy-kiali-operator.sh

```diff
@@ -1246,3 +1247,19 @@ EOF
 fi
 
 echo "Done."
+
+if [ "${AUTH_STRATEGY}" == "token" ]; then
+  CLIENT_BASENAME=$(basename ${CLIENT_EXE})
+  echo
+  echo "You are using \"token\" strategy for authentication. You will need to provide"
+  echo "a valid bearer token that your cluster will accept."
+  echo
+  echo "You could use the secrets of the Kiali ServiceAccount. To find the names of the secrets run:"
+  echo "  ${CLIENT_BASENAME} get serviceaccount -n ${NAMESPACE} kiali-service-account -o jsonpath='{.secrets[*].name}'"
+  echo
+  echo "If there are several secrets associated with the Kiali ServiceAccount, you will need to find which"
+  echo "one is accepted at login."
+  echo
+  echo "Get the bearer token of the secrets using the command:"
+  echo "  ${CLIENT_BASENAME} get secret -n ${NAMESPACE} {secret_name} -o jsonpath='{.data.token}'"
+fi
```

Comment:
I wouldn't put any of this in here. This should be in the docs. Because if we spit this out for "token", why not also spit out messages for "openid" (e.g. "You are using openid strategy. You will need to provide a valid OpenID Connect implemention in your cluster or things will not work. Yadda yadda yadda, etc, etc, etc").

So rather than have this long and loud output of this script (which only shows up once - if the user misses it, they miss it completely and its lost to them), we should put this in our docs.

Reply:
OK, I removed all this text. I'll do that change of the kiali/kiali repo.